tox.ini

```diff
@@ -3,14 +3,13 @@
 envlist = {py36,py39}-{dj32}
           {py38,py310}-{dj40}
           {py38,py310}-{dj41}
-
-toxworkdir = {homedir}/.tox-django-typed-models
+          {py38,py310}-{dj42}
 
 [testenv]
 changedir = {toxinidir}
 commands =
     pip --disable-pip-version-check install -e .
-    coverage run {envbindir}/py.test --ds=test_settings typedmodels/tests.py {posargs}
+    coverage run {envbindir}/pytest --ds=test_settings typedmodels/tests.py {posargs}
```

Comment:
The pytest command changed to `pytest` in 3.0.0 (2016): https://docs.pytest.org/en/7.1.x/changelog.html#release-3-0-0

(`py.test` still works but best to use the new one)